.github/workflows/pipeline.yml

```diff
@@ -28,11 +28,9 @@ jobs:
       - name: Install Composer
         uses: "ramsey/composer-install@v3"
         with:
+          composer-options: "--no-scripts"
```

Comment:
no scripts defined in the `composer.json`

Reply:
I just kept the option like before, shall I remove it?

Reply:
true, yes, can be removed :)